pkg/models/jsonschema/tag.go

```diff
@@ -12,6 +12,7 @@ import (
 type Tag struct {
 	Name          string        `json:"name,omitempty"`
 	Description   string        `json:"description,omitempty"`
+	Favorite      bool          `json:"favorite,omitempty`
```

Comment:
This is one of your errors.

Change to:  `json:"favorite,omitempty"` (you're forgetting the closing quotation)